tasks/Task.hpp

```diff
@@ -8,6 +8,12 @@
 #include <base/samples/RigidBodyState.hpp>
 #include <usbl_seatrac/Driver.hpp>
 
+/**
+ * The minimum ping refresh rate in case it will need the local usbl's orientation
+ * information
+ */
+static const base::Time MINIMUM_PING_STATUS_REFRESH_TIME = base::Time::fromSeconds(2.4);
+
```

Comment:
How did you get 2.4?

Reply:
I tested it in field. its the minimum refresh time possible that allows a correct orientation to be extracted from the ping.